trello_config.py

```diff
@@ -0,0 +1,17 @@
+"""Trello configuration class."""
+import os
+
+class Config:
+    """Base configuration variables."""
+    TRELLO_API_KEY = os.environ.get('TRELLO_API_KEY')
```

Comment:
These could become class attributes on a TrelloItems class in `trello_items.py` (avoiding you having to prepend `TRELLO_` to everything here)